eve/workers/openstack-multiple-nodes/terraform/scripts/network-iface-config.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
Can't we do all of this through `cloud-init`?

Reply:
Yes, that's exactly what the comment below says, I don't know yet how to do it through terraform, but I'll search.
The script is more to make the rest of the stuff work until I found the right solution.